physics/module_bl_mynn.F90

```diff
@@ -3086,9 +3090,10 @@ SUBROUTINE  mym_turbulence (                                &
        !qBUOY1D(k) = elq*(sh(k)*(-dTdz*grav/thl(k)) + gamv) !! ORIGINAL CODE
        
        !! Buoyncy term takes the TKEprodTD(k) production now
-       qBUOY1D(k) = elq*(sh(k)*gh(k)+gamv)+TKEprodTD(k) !staggered
+!fix?  qBUOY1D(k) = elq*(sh(k)*gh(k)+gamv)+TKEprodTD(k) !staggered
```

Comment:
ditto (Git history captures these changes. Of course, unless you have a reason for keeping these comments in here?)

Reply:
removed